src/test/java/com/kodilla/ecommercee/domain/GroupEntityTests.java

```diff
@@ -0,0 +1,86 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+@SpringBootTest
+@Transactional
+public class GroupEntityTests {
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Test
+    public void testSaveGroup() {
+
+        //Given
+        Group group = new Group();
+
+        Product product = new Product();
+        List<Product> productList = new ArrayList<>();
+        productList.add(product);
+        productRepository.save(product);
+        group.setProducts(productList);
+        group.setName("Test Group");
+
+        //When
+        groupRepository.save(group);
+        Long groupId = group.getGroupId();
+
+        //Then
+        Assertions.assertNotEquals(0, groupId);
+    }
+
+    @Test
+    public void testDeleteGroup() {
+        //Given
+        Group group = new Group();
+        group.setName("Ubrania");
+        Product product = new Product();
+        group.getProducts().add(product);
+        product.setGroup(group);
+
```

Comment:
I think you should standardize the creation of input data. In `testSavGroup` you have:
`List<Product> productList = new ArrayList<>();`
        `productList.add(product);`
       `productRepository.save(product);`
And in `testDeleteGroup` you used:` group.getProducts().add(product);` instead and didn't save product in repository. 
Please check the save product in repository is really needed and unify the code to clarify :)

Reply:
ok, i'll fix the delete. The rest I think are fine. Tests work :)